core/channel/push_combined_channel.hpp

```diff
@@ -45,7 +45,8 @@ class PushCombinedChannel : public Source2ObjListChannel<DstObjT> {
     ~PushCombinedChannel() override {
         ShuffleCombinerFactory::remove_shuffle_combiner(this->channel_id_);
 
-        this->src_ptr_->deregister_outchannel(this->channel_id_);
+        // TODO: what if src has already been released
+        // this->src_ptr_->deregister_outchannel(this->channel_id_);
```

Comment:
Don't know how to solve the problem at the moment, so I just turn this line into a comment.